lib/block_view_types.go

```diff
@@ -110,9 +110,14 @@ const (
 	EncoderTypeDmThreadEntry                     EncoderType = 37
 	EncoderTypeDeSoNonce                         EncoderType = 38
 	EncoderTypeTransactorNonceEntry              EncoderType = 39
+	EncoderTypeStateChangeEntry                  EncoderType = 40
```

Comment:
Do me a favor and let's start adding comments to these. You don't need to annotate the others, but for the new ones you added just mention how they're used.